lib/private/machines/create-record.js

```diff
@@ -103,22 +102,21 @@ module.exports = {
       // Otherwise, IWMIH we'll be sending back a record:
       // ============================================
 
-      // Sanity check: Verify that there is only one record.
-      if (nativeResult.ops.length !== 1) {
-        return exits.error(new Error('Consistency violation: Unexpected # of records returned from Mongo (in `.ops`).  Native result:\n```\n'+util.inspect(nativeResult, {depth: 5})+'\n```'));
-      }
-
       //  ╔═╗╦═╗╔═╗╔═╗╔═╗╔═╗╔═╗  ┌┐┌┌─┐┌┬┐┬┬  ┬┌─┐  ┬─┐┌─┐┌─┐┌─┐┬─┐┌┬┐
       //  ╠═╝╠╦╝║ ║║  ║╣ ╚═╗╚═╗  │││├─┤ │ │└┐┌┘├┤   ├┬┘├┤ │  │ │├┬┘ ││
       //  ╩  ╩╚═╚═╝╚═╝╚═╝╚═╝╚═╝  ┘└┘┴ ┴ ┴ ┴ └┘ └─┘  ┴└─└─┘└─┘└─┘┴└──┴┘
       // Process record (mutate in-place) to wash away adapter-specific eccentricities.
-      var phRecord = nativeResult.ops[0];
-      try {
-        processNativeRecord(phRecord, WLModel, s3q.meta);
-      } catch (e) { return exits.error(e); }
+      (function(iifeDone) { mongoCollection.findOne({ _id: nativeResult.insertedId }).then(function(phRecord) { iifeDone(undefined, phRecord);}).catch(function(err) { iifeDone(err);});})(function(err, phRecord) {
```

Comment:
same as https://github.com/balderdashy/sails-mongo/pull/499/files#r1402546571

Reply:
On the useNewUrlParser @mikermcneil that's no longer a supported config option in the newer version of the MongoDB driver. @eashaw can you confirm if that's the case for you?